src/common/POLFields/FieldsLocation/FieldsLocation.js

```diff
@@ -28,7 +28,18 @@ import {
 
 const NO_VALIDATE = [];
 
-const FieldsLocation = ({ locations, disabled, isDisabledToChangePaymentInfo, withValidation }) => {
+const FieldsLocation = ({
+  changeLocation,
+  disabled,
+  isDisabledToChangePaymentInfo,
+  locationIds,
+  locations,
+  withValidation,
+}) => {
+  const onChangeLocation = useCallback((location, name) => changeLocation(name, location.id), [changeLocation]);
```

Comment:
this doesn't make too much sense, just handle `location, fieldName` `onChangeLocation`